frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,165 @@
+      .should('be.visible', { timeout: 1 * MINUTE })
+      .invoke('text')
+      .should('eq', BOUND);
+  });
+
+  it('Test if owner and creation date are shown correctly', () => {
+    cy.byTestSelector('details-item-value__Owner')
+      .scrollIntoView()
+      .should('exist')
+      .invoke('text')
+      .should('eq', 'No owner');
+    cy.byTestSelector('details-item-value__Created at')
+      .scrollIntoView()
+      .should('exist')
+      .invoke('text')
+      .should('eq', 'a few seconds ago');
+  });
+
+  it('Test if secret data is masked', () => {
+    cy.contains('Reveal Values').should('exist');
+    cy.byTestID('co-copy-to-clipboard__text').as('secrets');
+    cy.get('@secrets').should(($el) => {
+      const childCount = $el.length;
+      const hiddenText = 'Value hidden';
+      const elText = $el.text();
+      const secrets = elText.split(hiddenText).slice(1);
+      secrets.forEach((arrEl: string) => {
+        expect(arrEl).to.equal(MASK);
+      });
+      expect(secrets.length).to.equal(childCount);
+    });
+  });
+
+  it('Test if secret data can be revealed', () => {
+    obcHandler.revealHiddenValues();
+    cy.byTestID('co-copy-to-clipboard__text').then(($el) => {
+      cy.byTestID('secret-data__dt').should(($h) => {
+        expect($h[0].innerText).to.equal('Endpoint');
+        expect($el[0].innerText).to.include(NS);
+        expect($h[2].innerText).to.equal('Access Key');
+        expect($el[2].innerText).to.match(new RegExp(ACCESS_KEY));
+        expect($h[3].innerText).to.equal('Secret Key');
+        expect($el[3].innerText).to.match(new RegExp(SECRET_KEY));
+      });
+    });
+  });
+
+  it('Test if secret data can be hidden again', () => {
+    obcHandler.hideValues();
+    cy.byTestID('co-copy-to-clipboard__text').as('secrets');
+    cy.get('@secrets').should(($el) => {
+      const childCount = $el.length;
+      const hiddenText = 'Value hidden';
+      const elText = $el.text();
+      const secrets = elText.split(hiddenText).slice(1);
+      secrets.forEach((arrEl) => {
+        expect(arrEl).to.equal(MASK);
+      });
+      expect(secrets.length).to.equal(childCount);
+    });
+  });
+
+  it('Test if labels and annotations are shown correctly', () => {
+    cy.byTestID('label-list').contains(NOOBAA_LABEL);
+    cy.byTestSelector('details-item-value__Annotations').contains(NO_ANNOTATIONS);
+  });
+
+  it('Test if namespace and secret are shown correctly', () => {
+    obcHandler.assertNamespaceExists();
+    cy.get('[title="Secret"]')
+      .siblings()
+      .contains(OBC_NAME);
+  });
+
+  it('Test if status and storage class are shown correctly', () => {
+    cy.byTestID('status-text').contains(BOUND);
+    cy.byLegacyTestID('openshift-storage.noobaa.io').contains(OBC_STORAGE_CLASS_EXACT);
+  });
+
+  it('Test if Object Bucket is created', () => {
+    const obName = `obc-${testName}-${OBC_NAME}`;
+    cy.contains(obName).click();
+    cy.byTestID('co-m-loader').should('not.exist');
```

Comment:
```suggestion
    detailsPage.isLoaded();
```
Present in `integration-tests-cypress/views/`